apps/cow-tools/src/app/milkman/[network]/order/new/page.tsx

```diff
@@ -0,0 +1,426 @@
+            type="string"
+            label="Token sell"
+            placeholder="0x.."
+            {...register("tokenSell")}
+          />
+        </div>
+        <div className="flex w-1/2 items-end gap-2">
+          <div className="w-full">
+            <Input
+              type="string"
+              label="Amount"
+              placeholder="0.0"
+              {...register("tokenAmount")}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="w-full">
+        <Input
+          type="string"
+          label="Token buy"
+          placeholder="0x.."
+          {...register("tokenBuy")}
+        />
+      </div>
+      <div>
+        <Input
+          type="string"
+          label="Recipient"
+          placeholder={userAddress}
+          {...register("receiverAddress")}
+        />
+        <div className="mt-2 flex gap-x-1 text-xs">
+          <button
+            type="button"
+            className="text-blue9 outline-none hover:text-amber9"
+            onClick={() => {
+              setValue("receiverAddress", userAddress);
+            }}
+          >
+            Use Current Address
+          </button>
+        </div>
+      </div>
+      <Controller
+        control={control}
+        name="isValidFromNeeded"
+        render={({ field: { onChange, value } }) => (
+          <Checkbox
+            id="isValidFromNeeded"
+            checked={value}
+            onChange={() => onChange(!value)}
+            label="Order will need valid from"
+          />
+        )}
+      />
+      {isValidFromNeeded && (
+        <div>
+          <Input
+            type="datetime-local"
+            label="Valid from"
+            {...register("validFrom")}
+          />
+          <div className="mt-2 flex gap-x-1 text-xs">
+            <button
+              type="button"
+              className="text-blue9 outline-none hover:text-amber9"
+              onClick={() => {
+                setValue("validFrom", formatDateToLocalDatetime(new Date()));
+              }}
+            >
+              Use Current Date time
+            </button>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function FormSelectPriceChecker({ form }: { form: UseFormReturn }) {
+  const { register, control } = form;
+  return (
+    <div className="flex flex-col gap-y-6 p-9">
+      <div className="mb-2">
+        <Label>Price checker</Label>
+        <Controller
+          control={control}
+          name="priceChecker"
+          render={({ field: { onChange, value } }) => (
+            <Select
+              onValueChange={onChange}
+              value={value}
+              className="w-full mt-2"
+            >
+              <SelectItem value="0xEB2bD2818F7CF1D92D81810b0d45852bE48E1502">
+                FixedMinOutPriceChecker
+              </SelectItem>
+            </Select>
+          )}
+        />
+      </div>
+      {/* //TODO get this when the priceChecker is selected */}
```

Comment:
We would have to get all input values right? Why just this one have this comment?

Reply:
I expressed myself wrong xD